lib/web_ui/dev/test_platform.dart

```diff
@@ -96,6 +96,7 @@ class BrowserPlatform extends PlatformPlugin {
         //  * /test/alarm_clock_test.dart
         //  * /lib/src/engine/alarm_clock.dart
         .add(createStaticHandler(env.environment.webUiRootDir.path))
+        .add(_createSourceHandler())
```

Comment:
Both `_createSourceHandler` and the previous handler attempt to handle `.dart` sources. Can't tell which one wins when.

Reply:
Turns out the first one is subsumed by the new one, so I removed the old one.

Reply:
Actually, we still need that static handler, because it serves specific files that the tests use, but it isn't for sources anymore, so I updated the comment.